src/routes/api/category-routes.js

```diff
@@ -0,0 +1,133 @@
+const router = require("express").Router();
+const { Category, Product } = require("../../models");
+
+// The `/api/categories` endpoint
+
+router.get("/", async (req, res) => {
```

Comment:
Although this is okay, it is best practice to separate the controller functions into a `controllers` folder. This keeps the routes clear and easy to follow.